docs/ROADMAP.md

```diff
@@ -0,0 +1,86 @@
+
+## Milestones
+
+Version numbering terminology:
+
+* **v0.1**: refers to the initial buildout that will be submitted to the app stores, aiming for Tuesday, March 24, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
+* **v0.2+**: any versions starting with major version v0 are not intended for public distribution or consumption
+* **v1.0**: the first version that we feel offers enough value to be published onto the app stores for public download will be v1.0
+
+### v0.1
+
+**Goal:** To get a functional build of an application that can be submitted to the Apple App Store and Google Play Store for approval. This app will _NOT_ be published into those app stores for public download, but will instead serve as an approved base for future versions and to prove out that we can deliver on the technical side. The app will serve static content manually adapted from the WHO's site in English.
+
+**Target Date:** Tuesday, March 24, 2020
+
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/2
+
+Technical Features / Requirements:
+
+* App built to be submitted on Apple App Store and Google Play Store, including code signing process
+* UI styled to exact [design spec in Figma](https://www.figma.com/file/fjzTIloCEK4FpbyDiTLj2X/iOS-UX) ([#186](https://github.com/WorldHealthOrganization/app/issues/186))
+  * Displaying static content in English only
+* App is still functional in an offline environment with default content able to be accessed without a network connection ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* Demonstrate capability to localize content from the technical platform perspective, even if other languages/cultures are _NOT_ part of that v0.1 launch
+  * Framework in place to localize UI elements ([#126](https://github.com/WorldHealthOrganization/app/pull/126) :white_check_mark:)
+  * Framework in place to load the informational content in localized formats ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* Demonstrate capability to load content dynamically from the cloud, even if this feature will _NOT_ be turned on as part of the v0.1 launch ([#120](https://github.com/WorldHealthOrganization/app/pull/120) :white_check_mark:)
+* _(Open Question)_ Demonstrate capability to instrument the app and collect basic analytics around which features and pages are actually being used, even if this feature will _NOT_ be turned on as part of the v0.1 launch ([#183](https://github.com/WorldHealthOrganization/app/issues/183))
+  * Investigating legal requirements of this data collection, including whether updated TOS/Privacy Policy is needed
```

Comment:
Focus on first words

You should create an issue for this and assign it to Michael Nichols (not sure of his GitHub) and @brunobowden so I'm subscribed on it as well. All the tasks for v0.1 should have issues created for them

```suggestion
  * Legal requirements investigate for data collection, including whether updated TOS/Privacy Policy is needed
```